### PR TITLE
Color picker fixes

### PR DESCRIPTION
**Pull request by djrm**

### CODE REVIEW

**Comment by djrm** [comment]
the HS case mentioned is the following

screenshot from 2016-07-01 16-27-07

File: scene/gui/color_picker.cpp [hunk 1]
```diff
@@ -174,22 +173,16 @@ void ColorPicker::_update_color() {
...
 		scroll[i]->set_step(0.01);
-		if (raw_mode_enabled && i != 3)
+		if (raw_mode_enabled) {
+			if (i == 3)
+				scroll[i]->set_max(1);
 			scroll[i]->set_val(color.components[i]);
-		else
-			scroll[i]->set_val(color.components[i]*255);
+		} else {
+			scroll[i]->set_val(color.components[i] * 255);
```
Comment (MarianoGnu) [inline scene/gui/color_picker.cpp]: you are doing it again, why do you multiply the compunent by 255? it's not suposed to be that way, only difference is allowing numbers avobe 1.

Comment (djrm) [inline scene/gui/color_picker.cpp]: @MarianoGnu this is the current godot behavior, its multiplied by that value because above 1 means from 0 to 255.

**Comment by akien-mga** [comment]
Needs rebase.

**Comment by djrm** [comment]
rebased

**Pull request merged**